projects/portfolio/portfolio-webpage-1.md

```diff
@@ -73,13 +73,13 @@ You should write clear + informative commit messages for each commit and create
 
 **Create your project**
 
-- [ ] Use bash to create a local portfolio-website directory. Use the command line to make an portfolio.html and an portfolio.css in the portfolio-website directory.
-- [ ] Use the command line to change the names of portfolio.html and portfolio.css to index.html and index.css.
-- [ ] Use the command line to make a gitignore file in your desktop directory and add at least one real unwanted file to the gitignore.
-- [ ] Use the command line to move the gitignore file from your desktop directory to your portfolio-website directory.
-- [ ] Using your command line, initialize a GitHub repo in portfolio-website.
-- [ ] Create a remote repo for the project in your GitHub account.
-- [ ] Using your command line, add and commit your portfolio-website directory, and then do a git push.
+- [ ] Use bash to create a local portfolio-website directory. Use the command line to make a portfolio.html file and a portfolio.css file in the portfolio-website directory.
+- [ ] Use the command line to change the names of portfolio.html and portfolio.css to index.html and styles.css.
+- [ ] Use the command line to make a .gitignore file in your desktop directory and add at least one real unwanted file to the .gitignore list.
+- [ ] Use the command line to move the .gitignore file from your desktop directory to your portfolio-website directory.
+- [ ] Using your command line, initialize a GitHub repo in portfolio-website directory.
+- [ ] Create a portfolio-website remote repository on Github. (DO NOT add a readME or .gitignore file in the set up)
```

Comment:
```suggestion
- [ ] Create a portfolio-website remote repository on Github. (DO NOT add a README or .gitignore file in the set up)
```

If you are adding the extra note about the README or .gitignore, it could be prudent to say why not

Reply:
@daaimah123 let me know if this looks good!
<img width="922" alt="Screenshot 2024-12-12 at 4 22 52 PM" src="https://github.com/user-attachments/assets/109b0cb7-8d50-4d32-8bed-8ea5656949cc" />

Reply:
Can you adjust this note to the effect of " * if * their local files already include a README and a .gitignore, they should not do ABC because XYZ"

Reply:
done! And, I added a visual via screenshot to make it more clear. @daaimah123